dashboard/app/models/levels/dsl_defined.rb

```diff
@@ -74,7 +74,9 @@ def self.create_from_level_builder(params, level_params, old_name = nil)
       # Save updated level data to external files
       if Rails.application.config.levelbuilder_mode
         level.rewrite_dsl_file(text)
-        rewrite_i18n_file(i18n)
+        # Only bother saving levels in scripts that we want to internationalize.
+        # As of August 2018, that's just the CSF levels.
+        rewrite_i18n_file(i18n) if level.script_levels.any? {|sl| sl.script.csf?}
```

Comment:
I think this is preventing proper save of non-CSF Multi levels?  I get a level with no strings:

![image](https://user-images.githubusercontent.com/413693/45060036-27b8d980-b053-11e8-904d-3e128ac33d8f.png)

I merged https://github.com/code-dot-org/code-dot-org/commit/a960487ff37644ad84aea6542960f90705a11d60 as a temporary fix.